src/stage1/ir.cpp

```diff
@@ -29544,6 +29544,23 @@ static IrInstGen *ir_analyze_instruction_check_switch_prongs(IrAnalyze *ira,
                     }
                 }
             }
+        } else {
```

Comment:
```suggestion
        } else if (!type_is_global_error_set(switch_type)) {
```

You can save a whole indentation level.